core/src/main/scala/za/co/absa/spline/core/listener/SplineQueryExecutionListener.scala

```diff
@@ -39,10 +40,16 @@ class SplineQueryExecutionListener(harvesterFactory: DataLineageBuilderFactory,
     if (funcName == "save") {
       log debug s"Start tracking lineage for action '$funcName'"
 
-      val builder = harvesterFactory.createBuilder(qe.sparkSession.sparkContext)
+      val rawLineage =
+        harvesterFactory.
+          createBuilder(qe.analyzed, Some(qe.executedPlan), qe.sparkSession.sparkContext).
```

Comment:
Upon merge to develop will method createBuilder needed? Seems like everything createBuilder could be just renamed to buildLineage. But right now you probably intend to reduce conflicts.

Reply:
Interesting. nitpick: logicalPlan could have stayed as buildLineage function parameter as it is used only there.